design/sd-local.md

```diff
@@ -171,26 +211,34 @@ The chart below shows relationship between `key` and `value`.
 |launcher-version|Version of Launcher Image (default: stable)|
 |launcher-image|Name of Launcher Image (default: screwdrivercd/launcher)|
 
-Create config file as `$HOME/.sdlocal/config` with YAML format.
-
-```yaml
-api-url: <Screwdriver API URL>
-store-url: <Screwdriver Store URL>
-token: <Screwdriver API Token>
-launcher:
-  version: <Launcher Version>
-  image: <Launcher image name>
-```
-
 Can confirm setting configurations with the command below:
 
 ```bash
 $ sdlocal config view
+*  cluster1:
+      api-url: https://cluster1-api-screwdriver.com
+      store-url: https://cluster1-store-screwdriver.com
+      token: hoge
+      launcher:
+        version: latest
+        image: screwdrivercd/launcher
+   cluster2:
+     api-url: https://cluster2-another-api-screwdriver.com
+     store-url: https://cluster2-another-store-screwdriver.com
+     token: fuga
+     launcher:
+       version: latest
+       image: screwdrivercd/launcher
```

Comment:
I think it looks like the indentations are not unified.

Reply:
fixed